var/spack/repos/builtin/packages/cube/package.py

```diff
@@ -36,6 +36,11 @@ class Cube(Package):
     homepage = "http://www.scalasca.org/software/cube-4.x/download.html"
     url = "http://apps.fz-juelich.de/scalasca/releases/cube/4.2/dist/cube-4.2.3.tar.gz"
 
+    # Some versions fail with parallel builds/installs
+    parallel = False
```

Comment:
For the latest versions I found it breaking for `make install` only, but earlier versions (4.2) AFAIR had problems with parallel builds as well. I don't like the global `parallel = False` here either - the build takes quite a lot of time. I guess to trigger it separately for `make/make install` you'll have to overwrite `install()` again?